src/platform_linux/shmem.c

```diff
@@ -517,11 +503,12 @@ platform_shm_alloc(platform_heap_id hid,
    }
 
    // Optimistically, allocate the requested 'size' bytes of memory.
-   retptr = __sync_fetch_and_add(&shminfop->shm_next, size);
+   _Static_assert(sizeof(void*)==sizeof(size_t), "check our casts are valid");
```

Comment:
Please run `clang-format` on this .. not sure if CI is running on this fix.

Reply:
Added a separate commit for this, since it fixed up other files too.